db/schema.rb

```diff
@@ -61,15 +61,15 @@
     t.text     "sub_title"
     t.text     "format"
     t.text     "details_url"
-    t.string   "publication_date"
-    t.string   "isbn"
-    t.string   "primary_language"
+    t.string   "publication_date",            limit: 255
```

Comment:
did you mean to put the string size limits?  this is a problem when the schema.db file is generated on a local system with some versions of postgres.  unfortunately, it's a major inconvenience.  iirc, there's a workaround, otherwise, we just have to make sure to not include the changes into a commit.

Reply:
Ok  I will remove it